### PR TITLE
Search and Results Component

### PR DESCRIPTION
**Pull request by Selchuck**
Description: 
Implemented Search bar for multiple criteria search( by location and programming language)
and display the search results in a tabular fashion.

Testing:
For eg : Search "Java" in the search language box and see the results in the table

### CODE REVIEW

File: src/frontend/components/Result/Results.js [hunk 1]
```diff
@@ -1,28 +1,37 @@
-import React from 'react';
-import './Results.css';
-import SearchBar from '../SearchBar/SearchBar';
-import Header from '../Header/Header';
-import people from '../../image/people.png';
+import React from "react";
```
Comment (DTFsquad) [inline src/frontend/components/Result/Results.js]: Please use a single quote in the imports as per Airbnb style guide and use double quotes just in the JSX part.

File: src/frontend/components/Result/Results.js [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
-export default function Results() {
-    return (
-        <div>
-            <SearchBar />
-            <table className="resultsTable">
+export function Results() {
+  return (
+    <div>
+      <SearchBar />
+       {/* <table className="resultsTable">
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/Result/Results.js]: comments can be removed

File: src/frontend/components/Result/Results.js [hunk 3]
```diff
@@ -1,28 +1,37 @@
...
+      <tr>
+        <td>
+          <img className="user-image" src={people} alt="people" />
+          <div>{props.name}</div>
+        </td>
+        <td>{props.language}</td>
+        <td>{props.location}</td>
+      </tr>
+      </tbody>
+    // </table>
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/Result/Results.js]: remove comments

File: src/frontend/components/SearchBar/SearchBar.js [hunk 4]
```diff
@@ -1,48 +1,138 @@
...
+              </tr>
+            </tbody>
+            {this.state.filteredUsers &&
+              this.state.filteredUsers.map(profile => profile)}
+          </table>
+        </div>
+        <div>{this.state.filteredLocation}</div>
+      </div>
+    );
+  }
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/SearchBar/SearchBar.js]: it is not ideal to have too many things in a file . Ideally this should be split into 3 files which we can do it later .Also destructuring the props. May be we can refactor in the next commit if you are out of time

**Pull request closed without merging**